src/codeSigning/SimplifiedSigning.ts

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
I'm assuming most of this code is how it was before? So I lightly skimmed it.

Reply:
discussd live, replying for posterity:  it's different.  Keys are generated during signing instead of passed in and are not persisted to fs.  

So the inputs and outputs are different.
It also combined the sign/upload/verify into a single exported function.  Now there's no way to upload a non-verifiable key, or sign but not upload.  That removes some non-viable potential states and makes it all-or-nothing. (then, CI retries make more sense).